catalyst_voices/packages/libs/catalyst_cardano/catalyst_cardano/example/test_driver/app_test.dart

```diff
@@ -0,0 +1,74 @@
+import 'package:catalyst_voices_driver/catalyst_voices_driver.dart';
+import 'package:flutter_driver/flutter_driver.dart';
+import 'package:test/test.dart';
+import 'package:webdriver/async_io.dart';
+
+///To run test driver you need to have chromedriver installed
+///
+///Then in terminal run:
+///chromedriver --port=4444
+///
+///In new terminal in example dir run:
+///
+///flutter drive --target=test_driver/app.dart \
+///--web-browser-flag=--disable-web-security \
+///--web-browser-flag=--disable-gpu \
+///--web-browser-flag=--disable-search-engine-choice-screen \
+///--web-header=Cross-Origin-Opener-Policy=same-origin \
+///--web-header=Cross-Origin-Embedder-Policy=require-corp \
+///--debug \
+///--no-headless \
+///-d web-server --browser-name=chrome --driver-port=4444
+void main() {
```

Comment:
```suggestion
/// To run test driver you need to have chromedriver installed
///
/// Then in terminal run:
/// ```sh
/// chromedriver --port=4444
/// ```
///
/// In new terminal in example dir run:
/// ```dart
///  flutter drive --target=test_driver/app.dart \
/// --web-browser-flag=--disable-web-security \
/// --web-browser-flag=--disable-gpu \
/// --web-browser-flag=--disable-search-engine-choice-screen \
/// --web-header=Cross-Origin-Opener-Policy=same-origin \
/// --web-header=Cross-Origin-Embedder-Policy=require-corp \
/// --debug \
/// --no-headless \
/// -d web-server --browser-name=chrome --driver-port=4444
/// ```
void main() {
```